README.md

```diff
@@ -49,12 +49,20 @@ docker pull gcr.io/prysmaticlabs/prysm/beacon-chain:latest
 First, clone our repository:
 
 ```
+sudo apt-get install git
 git clone https://github.com/prysmaticlabs/prysm
+cd prysm
 ```
 
 Download the Bazel build tool by Google here and ensure it works by typing:
 
 ```
+sudo apt-get install curl
```

Comment:
could you just add a link here to the bazel install page instead of having all these commands in the readme